torchdynamo/config.py

```diff
@@ -106,6 +106,13 @@
 # 3: Always dumps the last graph ran out to minify_repro.py, useful for segfaults/irrecoverable errors
 repro_level = int(os.environ.get("COMPILER_REPRO_LEVEL", 0))
 
+# TorchDynamo produced Fx GraphModule compilation debug info
+# 0: Nothing printed out when compilation fails
+# 1: Dump the initial graph to repro.tar.gz
+# 2/3: Minifies and Dumps the minified repro to repro.tar.gz
+dynamo_repro_level = int(os.environ.get("TORCHDYNAMO_REPRO_LEVEL", 0))
```

Comment:
Maybe we should call this AOT_AUTOGRAD_REPRO_LEVEL? This is mainly for minifying issues that occur at the AOTAutograd level, not the dynamo level.

Reply:
I earlier thought BACKEND_REPRO_LEVEL here, but then its confusing with COMPILER_REPRO_LEVEL.

We are going to use this for Aot Autograd, but others might use this for their backend. Is this any better?

* BACKEND_REPRO_LEVEL - Uses this PR
* INDUCTOR_REPRO_LEVEL 
* NVFUSER_REPRO_LEVEL

The last 2 flags set the `compiler_repro_flag` internally. Just for the user, this might be more clear?

Reply:
Or, just have COMPILER_REPRO_LEVEL as the user-level contract

COMPILER_REPRO_LEVEL = 0 : Do nothin
COMPILER_REPRO_LEVEL = 1 : Minifies the Backend
COMPILER_REPRO_LEVEL = 2 : Minifies for the compiler (exisiting one)

And do away with current leveling. We can have a separate flag REPRO_DETAIL for development purposes and fixing issues with minifier.